src/panoptes/pocs/focuser/birger.py

```diff
@@ -284,34 +310,6 @@ def _parse_move_response(self, response):
 
         return amount
 
-        # Get the version string of the adaptor software libray. Accessible as self.library_version
-        self._get_library_version()
-
-        # Get the hardware version of the adaptor. Accessible as self.hardware_version
-        self._get_hardware_version()
-
-        # Get basic lens info (e.g. '400mm,f28' for a 400 mm, f/2.8 lens). Accessible as
-        # self.lens_info
-        self._get_lens_info()
-
-        # Initialise the aperture motor. This also has the side effect of fully opening the iris.
-        self._initialise_aperture()
-
-        # Initalise focus. First move the focus to the close stop.
-        self._move_zero()
-
-        # Then reset the focus encoder counts to 0
-        self._zero_encoder()
-        self._min_position = 0
-
-        # Calibrate the focus with the 'Learn Absolute Focus Range' command
-        self._learn_focus_range()
-
-        # Finally move the focus to the far stop (close to where we'll want it) and record position
-        self._max_position = self._move_inf()
-
-        self.logger.info('{} initialised'.format(self))
```

Comment:
Was this intentionally removed?

Reply:
The logging call? Yep - there is an equivalent call in the serial base class.